### PR TITLE
[hue] Channel alert added for groups

### LINKED ISSUE #7742: [hue] add channel for action/alert
@lolodomo 
So glad to discover that support for HUE groups have been added.
Up until now I have relied on the http binding sending actions to HUE groups via the REST API.

Before I can burn all bridges though, I would really appreciate if you could include a channel for the action/alert. I use that a lot notifying my disabled mother when the door chimes, or telephone rings.

This feature is really nice, because the lights resume their initial state after done blinking.

I have found that when you do a http GET request for a HUE group over REST, only the original HUE hub return the alert action in the reply, but deconz does not.
It works just fine on deconz as well though. I just tested it on my deconz on a group of 2 HUE strips and 2 Ikea Floalt panels. Here is the REST reply:

```
2020-05-22 18:15:48.899 [INFO ] [clipse.smarthome.model.script.deconz] - deconzAlarm triggerd by vAllLightsAlarm=ON
2020-05-22 18:15:48.910 [INFO ] [clipse.smarthome.model.script.deconz] - sendHttpPutRequest() returned: [{"success":{"/groups/3/action/alert":"lselect"}}]
```
The OH2 channel should accept 2 alarm types, and the ability to abort:

NONE, SELECT and LSELECT

Thanks.

### PR DESCRIPTION
**Pull request by lolodomo**
Fix #7742

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

File: bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueGroupHandler.java [hunk 1]
```diff
@@ -377,4 +403,68 @@ public void onGroupGone(@Nullable HueBridge bridge, FullGroup group) {
...
+                updateState(CHANNEL_ALERT, new StringType(LightStateConverter.ALERT_MODE_NONE));
+            }, delay, TimeUnit.MILLISECONDS);
+        }
+    }
+
+    /**
+     * This method will cancel previously scheduled alert item state
+     * restoration.
+     */
+    private void cancelScheduledFuture() {
```
Comment (cpmeister) [inline bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueGroupHandler.java:436]: You should call this when disposing this handler as well.

Comment (lolodomo) [reply bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueGroupHandler.java:436]: Yes, you're right.
I have a copy/paste from the hue light handler and this is missing in this handler too.

Comment (lolodomo) [reply bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueGroupHandler.java:437]: Fixed

**Comment by olemr** [comment]
Should the documentation be updated with **group** as a supported thing type?
image

**Pull request merged**